### PR TITLE
Skullcrush instakill hotfix

### PR DESCRIPTION
**Pull request by saltprjak**
## About The Pull Request

Turns out the "mortal" flag was bugged, and was not taken into account during interactions. Something about the SOUL PR (currently testmerged) suddenly made it work, which kicked in another piece of bad code from before where it checks whether the victim has a mortal-flagged fracture OR critical weakness, not both at the same time. I can't really test this alongside SOUL PR, but this should fix instances where head fractures caused by blunt attacks and stabs instantly kill you, regardless of critical weakness. Some pieces could use a rewrite, but i can't be bothered

## Why It's Good For The Game

better call saulgood for the game man

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by ThePainkiller** [comment]
This PR will make maces physically incapable of kill anyone again, mind you, because these wounds are specific to eye, ears, and top of the head (skull) crits, not overall general head target fractures.

**Comment by 1N7Slayer** [comment]
> This PR will make maces physically incapable of kill anyone again, mind you, because these wounds are specific to eye, ears, and top of the head (skull) crits, not overall general head target fractures.

I'd say leave 'mortal = TRUE` on the top of the skull/cranium. Armor protects the most there, as opposed to the eyes, which people game a lot by just targeting eyes right now. If people wanna fish for instant permakills they should have to aim for the skull.

**Comment by ThePainkiller** [comment]
>I'd say leave 'mortal = TRUE` on the top of the skull/cranium. Our in-game armor items protects the most there, as opposed to the eyes, which people game a lot by just targeting eyes right now. If people wanna fish for instant permakills they should have to aim for the skull.

Not a bad idea, as a fix for now
My goal was, at a later date and different PR, to make it so that a broken skull was just one catch-all type of wound, that added a trait of weakness to the person in question (maybe a trait? Maybe a var flipped to TRUE?), so that skull targetted mace crits would check for this, and if true, cause death instead due to the weakness of having a literal exposed brain. This could patch things for now so that these wounds aren't lethal anymore

**Comment by Cre77** [comment]
> This PR will make maces physically incapable of kill anyone again, mind you, because these wounds are specific to eye, ears, and top of the head (skull) crits, not overall general head target fractures.

but instakills are a VERY bad way to go about it. they couldn't "kill anyone" before this to begin with, before my KO PR they still performed the exact same. this just fixes an oversight that made them arguably DEADLIER THAN SWORDS in that regard, since they're not tied to sharpness when it comes to skullcrushing people's brains out.
no one likes an instakill, and no one especially likes an instakill while apprehending because they fatfingered the target area and chose the wrong piece of head. and personally? i play around with maces a lot, both as barb and as various guard roles - including the high STR captain - and i feel like they're in a good place. the knockdowns you get after a certain breakpoint are an immense help in teamfights or CC, and the paincrit/disabling from fractures helps take down people safely without bloodshed, which usually led to nothing but improvements all around.
PLEASE consider plugging this PR in as-is, instakills should be very hard to achieve by player means, and with very deliberate effort. stonekeep is VERY player interaction heavy, and that should be nurtured

**Comment by ThePainkiller** [comment]
> tl;dr - mace good, instakill bad, get good

Sorry if you missunderstood me, the PR is good to go, but it will be a temporary bandaid fix that will bring back maces down to SS13 stunbaton levels of lethality, just as a warning
I'll get to work on my cummulative wounds PR at a later date and make it so that having a broken skull makes the next crit to instantly kill you.

**Comment by Cre77** [comment]
> > tl;dr - mace good, instakill bad, get good
> 
> Sorry if you missunderstood me, the PR is good to go, but it will be a temporary bandaid fix that will bring back maces down to SS13 stunbaton levels of lethality, just as a warning I'll get to work on my cummulative wounds PR at a later date and make it so that having a broken skull makes the next crit to instantly kill you.

actually incredible piece of game design, the latter, i love it

**Review by ThePainkiller — APPROVED** [review]

**Comment by Eonoc** [comment]
https://github.com/Darkrp-community/OpenKeep/issues/814#issue-2553690278
